shortex.sty

```diff
@@ -146,17 +149,17 @@
 
 % Brackets and bracket-like functions
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\newcommand{\rbra}[2][{-1}]{\lcrx[#1] ( {#2} ) }
-\newcommand{\cbra}[2][{-1}]{\lcrx[#1] \{ {#2} \} }
-\newcommand{\sbra}[2][{-1}]{\lcrx[#1] [ {#2} ] }
-
-\newcommand{\abs}[2][{-1}]{\lcrx[#1] \vert {#2} \vert }
-\newcommand{\set}[2][{-1}]{\lcrx[#1] \{ {#2} \}}
-\newcommand{\floor}[2][{-1}]{\lcrx \lfloor {#2} \rfloor}
-\newcommand{\ceil}[2][{-1}]{\lcrx[#1] \lceil {#2} \rceil}
-\newcommand{\norm}[2][{-1}]{\lcrx[#1] \Vert {#2} \Vert}
-\newcommand{\inner}[3][{-1}]{\lcrx[#1] \langle {{#2},\ {#3}} \rangle}
-\newcommand{\card}[2][{-1}]{\lcrx[#1] \vert {#2} \vert }
+\newcommand{\rbra}[2][{-2}]{\lcrxx[#1] ( {#2} ) }
+\newcommand{\cbra}[2][{-2}]{\lcrxx[#1] \{ {#2} \} }
+\newcommand{\sbra}[2][{-2}]{\lcrxx[#1] [ {#2} ] }
+
+\newcommand{\abs}[2][{-1}]{\lcrxx[#1] \vert {#2} \vert }
```

Comment:
Bracket-like notation that was defined by lcrx before and is defined by lcrxx now has default argument -1 instead of -2 (\abs ... \card lines 152-162). 
Is this on purpose?
I think they should use -2 to have the same behaviour \rbra, \cbra,...